torch/testing/_internal/common_methods_invocations.py

```diff
@@ -16783,6 +16783,17 @@ def sample_inputs_alias_copy(op_info, device, dtype, requires_grad, **kwargs):
            skips=(
                # Resized a non-empty tensor but did not warn about it
                DecorateInfo(unittest.expectedFailure, 'TestCommon', 'test_out_warning'),
+               # FIXME:
+               # Expected 2114 but got 1123.
+               # Absolute difference: 991 (up to 0.001 allowed)
+               # Relative difference: 0.46877956480605487 (up to 0.001 allowed)
+               DecorateInfo(
+                   unittest.skip("Skipped!"),
```

Comment:
Why not xfail here?

Reply:
I think there's some risk of this test passing depending on the GPU hardware, so I did skip to avoid unexpected successes. I can try xfail instead